test/app.js

```diff
@@ -169,3 +169,37 @@ describe('generator-camel:app', function () {
     });
   });
 });
+
+describe('generator-camel:wsdl2rest', function () {
+  describe('Should properly scaffold with config for Spring and wsdl2rest', function () {
+    it('Should create the basic structure two ways', function () {
+      basicProps.name = 'MyAppMock';
+      basicProps.package = 'com.generator.mock';
+      basicProps.camelVersion = '2.18.2';
```

Comment:
any reasons to choose this "old" version? What about using the latest released at the moment 2.22.2? or we want to be backward-compatible?

Reply:
Will update to 2.22.2

Reply:
updated in https://github.com/camel-tooling/generator-camel-project/pull/33